pages/blog/dont-forget-your-vpn.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Don’t Forget Your VPNs and Tracker Blockers"
+date: "2021-01-27"
+author: "Andy Jih"
+pullQuote: "Ad blockers are great, until they’re not"
+image: "dont-forget-your-vpn/intercom_blocked.png"
+tags: [product, notes]
+---
+
+I was testing the new [Intercom](https://intercom.com) plugin that our team had built, and ran into a strange snag. When adding apps to Grouparoo, we have this handy “Test Connection” button that checks to make sure the connection details and credentials work. When I was testing this new Intercom plugin, something strange appeared this time around.
```

Comment:
Is there anywhere on our site we talk about intercom to link to?

Reply:
Not yet-- I'll update the link when that page is up!